### ISSUE TITLE
stream the attachment contents

### ISSUE THREAD

**Issue by dongbin86** [jonhoo/rust-imap#294]
I'm using this library to read email content. The attachments might be numerous and large, and I want to stream the attachment contents and save them gradually to disk, rather than loading everything into memory at once. Does the library support this approach? If so, do you have an example?

**Comment by jonhoo** [comment]
No, unfortunately there isn't support for this at the moment. You _could_ implement it manually by using the readline command that's exposed though. It'd be a good feature to add, though also non-trivial given the library's current design.
